File: fleeman/src/main/java/com/example/demo/entities/Customer.java
package com.example.demo.entities;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

import java.time.LocalDate;

@Entity
public class Customer {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int customerId;

    private String firstName;

    private String lastName;

    private String addressLine1;

    private String addressLine2;

    @Column(unique = true)
    private String email;

    private String city;

    private String pincode;

    private String phoneNumber;

    private String mobileNumber;

    private String creditCardType;

    private String creditCardNumber;

    private String drivingLicenseNumber;

    private String idpNumber;

    private String issuedByDL;

    private LocalDate validThroughDL;

    private String passportNumber;

    private LocalDate passportValidThrough;

    private String passportIssuedBy;

    private LocalDate passportValidFrom;

    private LocalDate passportIssueDate;

    private LocalDate dateOfBirth;
    
    private String password;
    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    
    public Customer() {
    }

    
    public int getCustomerId() {
        return customerId;
    }

    public void setCustomerId(int customerId) {
        this.customerId = customerId;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getAddressLine1() {
        return addressLine1;
    }

    public void setAddressLine1(String addressLine1) {
        this.addressLine1 = addressLine1;
    }

    public String getAddressLine2() {
        return addressLine2;
    }

    public void setAddressLine2(String addressLine2) {
        this.addressLine2 = addressLine2;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getPincode() {
        return pincode;
    }

    public void setPincode(String pincode) {
        this.pincode = pincode;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public String getMobileNumber() {
        return mobileNumber;
    }

    public void setMobileNumber(String mobileNumber) {
        this.mobileNumber = mobileNumber;
    }

    public String getCreditCardType() {
        return creditCardType;
    }

    public void setCreditCardType(String creditCardType) {
        this.creditCardType = creditCardType;
    }

    public String getCreditCardNumber() {
        return creditCardNumber;
    }

    public void setCreditCardNumber(String creditCardNumber) {
        this.creditCardNumber = creditCardNumber;
    }

    public String getDrivingLicenseNumber() {
        return drivingLicenseNumber;
    }

    public void setDrivingLicenseNumber(String drivingLicenseNumber) {
        this.drivingLicenseNumber = drivingLicenseNumber;
    }

    public String getIdpNumber() {
        return idpNumber;
    }

    public void setIdpNumber(String idpNumber) {
        this.idpNumber = idpNumber;
    }

    public String getIssuedByDL() {
        return issuedByDL;
    }

    public void setIssuedByDL(String issuedByDL) {
        this.issuedByDL = issuedByDL;
    }

    public LocalDate getValidThroughDL() {
        return validThroughDL;
    }

    public void setValidThroughDL(LocalDate validThroughDL) {
        this.validThroughDL = validThroughDL;
    }

    public String getPassportNumber() {
        return passportNumber;
    }

    public void setPassportNumber(String passportNumber) {
        this.passportNumber = passportNumber;
    }

    public LocalDate getPassportValidThrough() {
        return passportValidThrough;
    }

    public void setPassportValidThrough(LocalDate passportValidThrough) {
        this.passportValidThrough = passportValidThrough;
    }

    public String getPassportIssuedBy() {
        return passportIssuedBy;
    }

    public void setPassportIssuedBy(String passportIssuedBy) {
        this.passportIssuedBy = passportIssuedBy;
    }

    public LocalDate getPassportValidFrom() {
        return passportValidFrom;
    }

    public void setPassportValidFrom(LocalDate passportValidFrom) {
        this.passportValidFrom = passportValidFrom;
    }

    public LocalDate getPassportIssueDate() {
        return passportIssueDate;
    }

    public void setPassportIssueDate(LocalDate passportIssueDate) {
        this.passportIssueDate = passportIssueDate;
    }

    public LocalDate getDateOfBirth() {
        return dateOfBirth;
    }

    public void setDateOfBirth(LocalDate dateOfBirth) {
        this.dateOfBirth = dateOfBirth;
    }
}

File: pro3/Models/Customer.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace pro3.Models
{
    public class Customer
    {
        [Key]
        [Column(name: "Customer_Id")]
        public int CustomerId { get; set; }

        [Required]
        [Column(name: "First_Name")]
        public string FirstName { get; set; }

        [Required]
        [Column(name: "Last_Name")]
        public string LastName { get; set; }

        [Column(name: "Address_Line1")]
        public string AddressLine1 { get; set; }

        [Column(name: "Address_Line2")]
        public string AddressLine2 { get; set; }

        [Required]
        [EmailAddress]
        [Column(name: "Email")]
        public string Email { get; set; }

        [Column(name: "City")]
        public string City { get; set; }

        [Column(name: "Pincode")]
        public string Pincode { get; set; }

        [Column(name: "Phone_Number")]
        public string PhoneNumber { get; set; }

        [Column(name: "Mobile_Number")]
        public string MobileNumber { get; set; }

        [Column(name: "Credit_Card_Type")]
        public string CreditCardType { get; set; }

        [Column(name: "Credit_Card_Number")]
        public string CreditCardNumber { get; set; }

        [Column(name: "Driving_License_Number")]
        public string DrivingLicenseNumber { get; set; }

        [Column(name: "Idp_Number")]
        public string IdpNumber { get; set; }

        [Column(name: "Issued_By_DL")]
        public string IssuedByDL { get; set; }

        [Column(name: "Valid_Through_DL")]
        public DateTime ValidThroughDL { get; set; }

        [Column(name: "Passport_Number")]
        public string PassportNumber { get; set; }

        [Column(name: "Passport_Valid_Through")]
        public DateTime PassportValidThrough { get; set; }

        [Column(name: "Passport_Issued_By")]
        public string PassportIssuedBy { get; set; }

        [Column(name: "Passport_Valid_From")]
        public DateTime PassportValidFrom { get; set; }

        [Column(name: "Passport_Issue_Date")]
        public DateTime PassportIssueDate { get; set; }

        [Column(name: "Date_Of_Birth")]
        public DateTime DateOfBirth { get; set; }

        [Required]
        [Column(name: "Password")]
        public string Password { get; set; }
    }
}
